src/sort-files.ts

```diff
@@ -49,8 +49,26 @@ export default function sortFiles(packageJson: PackageJson): { files?: PackageJs
 
   const isPackageMain = (filepath: string) => filepath === main;
   const ignored = or(ALWAYS_INCLUDED, ALWAYS_EXCLUDED, isPackageMain);
+  const directoriesFirst = (a: any, b: any) => {
+    if (a.endsWith('/') && !b.endsWith('/')) {
+      return -1;
+    } else if (!a.endsWith('/') && b.endsWith('/')) {
+      return 1;
+    } else {
+      return a - b;
+    }
+  };
+  const exclusionsLast = (a: any, b: any) => {
```

Comment:
```suggestion
  const exclusionsLast = (a: string, b: string) => {
```

Reply:
Thanks for the suggestions @cameronhunter :+1: I've applied the changes.